components/zip.js

```diff
@@ -0,0 +1,32 @@
+// zip.js
+
+var _ = require('underscore');
+
+var wrapper = require('../src/javascript-wrapper.js');
+
+/**
+ * Merges together one of the values of the array with the given value
+ * @param array Array of values to choose from
+ * @param value to include in output
+ * @return an Array consistent of an item from array and value
```

Comment:
Why not just return an object directly, instead of returning a two-item array that is later turned into an object on line 31 of load-balancer.fbp?